templates/feedback.html

```diff
@@ -61,6 +76,59 @@ <h2 class="govuk-fieldset__heading">
     <input type="hidden" name="satisfaction" value={{form.satisfaction.value}}>
   {%  endif %}
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half-from-desktop">
+    <div class="govuk-form-group{% if form.experienced_issues.errors %} govuk-form-group--error{% endif %}">
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+          <h2 class="govuk-fieldset__heading">
+            {{ form.experienced_issues.label   }}
+          </h2>
+        </legend>
+        {% form_field_error form "experienced_issues" %}
+        <div class="govuk-checkboxes govuk-checkboxes--small" data-module="govuk-checkboxes" id="experienced_issues">
+          {% for value, name in form.fields.experienced_issues.choices %}
+            <div class="govuk-checkboxes__item">
+                <input class="govuk-checkboxes__input"
+                 id="{{ form.experienced_issues.name }}-{{ value }}"
+                 name="{{ form.experienced_issues.name }}"
+                 type="checkbox" value="{{ value }}"
+                 {% if value in form.experienced_issues.value %} checked="checked"{% endif %}
+                 data-aria-controls="conditional-experienced_issues-{{ value}}">
+                <label class="govuk-label govuk-checkboxes__label" for="{{ form.experienced_issues.name }}-{{ value }}">{{ name }}</label>
+            </div>
+
+            {% if forloop.counter == 5 %}
+
+            <div class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" id="conditional-{{form.experienced_issues.name}}-OTHER">
+                <div class="govuk-form-group">
+                    <div id="{{ form.other_detail.name }}-form-group" class="govuk-character-count" data-module="govuk-character-count" data-maxlength="1250">
+                        <div  class="govuk-form-group{% if form.update.errors %} govuk-form-group--error{% endif %}">
+                            <label class="govuk-label govuk-label--s" for="other_detail">{{ form.other_detail.label }}</label>
+                            {% form_field_error form "other_detail" %}
+                            <span class="govuk-hint">{{ form.other_detail.help_text }}</span>
+                            <textarea id="{{ form.other_detail.name }}"
+                            class="govuk-textarea govuk-js-character-count js-character-count"
+                            name="{{ form.other_detail.name }}"
+                            rows="5"
+                            maxlength="1250">{% if form.other_detail.value %}{{ form.other_detail.value}}{% endif %}</textarea>
```

Comment:
can use the built-in django template filter

```suggestion
                            maxlength="1250">{{ form.other_detail.value|default:''}}</textarea>
```

Reply:
Done